src/main/java/io/papermc/lib/environments/PaperEnvironment.java

```diff
@@ -11,7 +11,7 @@
     public PaperEnvironment() {
         super();
 
-        if (isVersion(13, 1)) {
+        if (isVersion(13, 1) || isVersion(14, 3)) {
```

Comment:
The async chunk api is still a [placeholder](https://github.com/PaperMC/Paper/blob/ver/1.14/Spigot-Server-Patches/0384-Async-Chunk-placeholder.patch) so I wouldn't mark that as included.

Reply:
I'm not expecting this to be accepted any time soon, I'm just putting this here for when the time comes. This stems from [slow render can't keep up. #2535](https://github.com/webbukkit/dynmap/issues/2535).